src/_includes/examples/animated.html

```diff
@@ -9,11 +9,11 @@ <h2 class="page-header">
   <div class="row">
     <div class="col-md-3 col-sm-4">
       <p>
-        <i class="fa fa-spinner fa-spin fa-3x fa-fw margin-bottom"></i>
-        <i class="fa fa-circle-o-notch fa-spin fa-3x fa-fw margin-bottom"></i>
-        <i class="fa fa-refresh fa-spin fa-3x fa-fw margin-bottom"></i>
-        <i class="fa fa-cog fa-spin fa-3x fa-fw margin-bottom"></i>
-        <i class="fa fa-spinner fa-pulse fa-3x fa-fw margin-bottom"></i>
+        <i class="fa fa-spinner fa-spin fa-3x fa-fw margin-bottom" aria-hidden="true"></i>
```

Comment:
This list of icons is communicating that something is busy, they should have alt texts